{
	"id": "CVE-2020-28275",
	"sourceIdentifier": "vulnerabilitylab@mend.io",
	"vulnStatus": "Rejected",
	"published": "2020-12-29T17:15:12.457",
	"lastModified": "2023-11-07T03:21:09.213",
	"descriptions": [
		{
			"lang": "en",
			"value": "Rejected reason: DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Reason: This candidate was withdrawn by its CNA. Further investigation showed that it was not a security issue. Notes: none"
		}
	],
	"references": [],
	"metrics": {}
}
